.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: ruby
+
+rvm:
+- 2.2
+- 2.1
+- 2.0.0
+- 1.9.3
+- 1.8.7
+- jruby-19mode
+- rbx-2
+
+sudo: false
+
+cache: bundler
+
+after_success:
```

Comment:
Could/should we use the `deploy` phase here? http://docs.travis-ci.com/user/customizing-the-build/#Deploying-your-Code